File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/Measures/Measure.cs
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.MeasureDetails;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.MeasureDetails;
using Ohtap.ECommerce.Domain.Models.Objects.Measures;
using Ohtap.ECommerce.Domain.Models.Objects.Units;
namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.Measures
{
    public class Measure : MeasureBasic, IMeasure
    {
        public IUnitDataList Units { get; set; }
        IMeasureDetailDataList IMeasure.Details { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Measures/Measure.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Measures;

class Measure  implements IMeasure
{

    public $Units;//UnitList
    public $Details;//MeasureData
    public $Description;
    public $State;
    public $OrderNo;
    public $Title;
    public $Name;
    public $ID;
}